packages/bot/src/Bot.ts

```diff
@@ -150,11 +150,17 @@ export class Bot extends events.EventEmitter {
         this.clientFunctions.push(func);
     }
 
-    addCommand(cmd: Command) {
+    /**
+     * @summary registers a {@link Command}
```

Comment:
I think comment unnecessary

Reply:
I am used to JSDoc'ing all methods/function signatures, but the comment is actually incomplete and thus looks off, I forgot the `@param cmd {@link Command} to register`. However, if you prefer not to add JSDoc, can remove, ofc

Reply:
I think ts documents most things fine, don't think it's needed

Reply:
No problem with that (just my default preference at play) - removed those JSDoc comments